node/cmd/guardiand/node.go

```diff
@@ -245,12 +245,11 @@ func runNode(cmd *cobra.Command, args []string) {
 		os.Exit(1)
 	}
 
-	cfg := zap.NewDevelopmentConfig()
-	cfg.Level = zap.NewAtomicLevelAt(zapcore.Level(lvl))
-	logger, err := cfg.Build()
-	if err != nil {
-		panic(err)
-	}
+	logger := zap.New(zapcore.NewCore(
+		consoleEncoder{zapcore.NewConsoleEncoder(
+			zap.NewDevelopmentEncoderConfig())},
+		zapcore.AddSync(zapcore.Lock(os.Stderr)),
+		zap.NewAtomicLevelAt(zapcore.Level(lvl))))
 
```

Comment:
This exact same diff is [here](https://github.com/certusone/wormhole/pull/744/files#diff-7f43e1a5bbb6083b5e470710e19e804d778f4a411b23fdbef1b2955f56b67bbdL248-R252) in #744. I think you messed up this spr stack